### PR TITLE
Mou is not macOS Sierra compatible

### PR DESCRIPTION
**Pull request by franklouwers**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Mou won't run on macOS Sierra:
objc[20930]: Objective-C garbage collection is no longer supported.

### CODE REVIEW

File: Casks/mou.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@
   homepage 'http://25.io/mou/'
   license :commercial
 
+  depends_on macos: '< :sierra'
```
Comment (reitermarkus) [inline Casks/mou.rb]: Could you change this to `'<= :el_capitan'`? We usually prefer `<=` and `>=`.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/mou.rb (hunk 1 above)
Comment (franklouwers) [inline Casks/mou.rb]: Changed in 7ba8c14

**Review by reitermarkus — APPROVED** [review]

**Comment by reitermarkus** [comment]
Thanks!

By the way, I'm a Mou user myself, did you find an alternative yet?

**Comment by jawshooah** [comment]
@reitermarkus I've used [MacDown](http://macdown.uranusjr.com/) before and been generally happy with it.

**Comment by reitermarkus** [comment]
@jawshooah, thanks, it even has the same split-view, probably because:

> It is heavily influenced by Chen Luo’s Mou.

**Comment by franklouwers** [comment]
I'm trying typora atm. But it has no split-screen

**Comment by vitorgalvao** [comment]
iA Writer is the one for me.

**Pull request merged**